schemas/pe.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: pe
+  title: PE Header
+  group: 2
+  description: These fields contain Windows Portable Executable (PE) metadata.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - process
+  fields:
+
+    - name: original_file_name
+      format: string
+      level: extended
+      type: keyword
+      description: Internal name of the file, provided at compile-time.
+      example: MSPAINT.EXE
+
+
+    - name: file_version
```

Comment:
If there's no other version attribute in PE, I think it would make sense to simply call this `pe.version` instead of `pe.file_version`, WDYT?

Reply:
there are other version attributes within PE headers, but are typically not collected and have less utility. this is a very narrow subset of all of the fields within PE headers, but does include the endgame and sysmon fields that are usable for detection logic. the fields I enumerated more closely resemble the original names, and if we rename we run the risk of causing more confusion

Reply:
Gotcha, let's keep it this way, then.